.github/workflows/windows.yml

```diff
@@ -111,4 +111,4 @@ jobs:
           cabal v2-run cabal-install:unit-tests -- --pattern "! (/FileMonitor/ || /VCS/ || /Get/)"
       - name: cabal-tests
         # Using only one job, -j1, to fail less.
-        run: cabal v2-run cabal-testsuite:cabal-tests -- -j1 --with-cabal=dist-newstyle\build\x86_64-windows\ghc-8.10.4\cabal-install-3.6.0.0\x\cabal\build\cabal\cabal.exe
+        run: cabal v2-run cabal-testsuite:cabal-tests -- -j1 --with-cabal=$(cabal list-bin exe:cabal)
```

Comment:
The windows executor is powershell, so bourne shell stuff doesn't work.

Reply:
Yeah, just noticed. Tried something for powershell, if that still doesn't work I'll just resort to not improving this.

Reply:
Please make improvements in separate commits. It would be great to have "this is how you bump version" commits, which don't do anything else.